### PR TITLE
Add Scoop installation methods for Windows

### PR DESCRIPTION
**Pull request by Zliced13**
### Motivation:

I wanted to add scoop installation methods for swift, since I am the one who added it to [scoop](https://scoop.sh/).

### Modifications:

I added scoop installation methods to these files, **getting-started/_installing.md** and the **download/_windows.md**

### Result:

**getting-started/_installing.md** and **download/_windows.md** will now have sections that detail how to install the dependencies and the Swift programming language using scoop.

### CODE REVIEW

**Comment by Zliced13** [comment]
This PR relates to #157.

**Comment by Zliced13** [comment]
@kaishin It's ready now.

**Comment by tomerd** [comment]
@compnerd to approve

File: getting-started/_installing.md [hunk 1]
```diff
@@ -87,24 +97,42 @@ The [Windows Package Manager](https://docs.microsoft.com/windows/package-manager
...
    * Using the official installer:
      1. Download the [latest package release](/download).
-     1. Run the package installer.
+     2. Run the package installer.
 
    * Using the Windows Package Manager:
      ~~~ cmd
      winget install Swift.Toolchain
      ~~~
+   * Using the Windows Package Manager:
```
Comment (compnerd) [inline getting-started/_installing.md:132]: I think that this should be "Using Scoop"?

```suggestion
   * Using Scoop:
```

File: getting-started/_installing.md [hunk 2]
```diff
@@ -71,11 +71,21 @@ In order to develop applications, particularly with the Swift Package Manager, y
...
+~~~ pwsh
+# Optional: Needed to run a remote script the first time
+Set-ExecutionPolicy RemoteSigned -Scope CurrentUser
+# Command for installing scoop
+Invoke-RestMethod -Url 'get.scoop.sh' | Invoke-Expression
+~~~
+
 0. Install required dependencies:
 
    The platform dependencies cannot be installed through the Windows Package Manager as the install rules do not install the components necessary.  They will be installed through Visual Studio installer.
```
Comment (compnerd) [inline getting-started/_installing.md:85]: This seems a bit confusing now with two options.

```suggestion
   The platform dependencies cannot be installed through the currently supported package managers as the install rules do not install the components necessary.  They will be installed through Visual Studio installer.
```

File: getting-started/_installing.md [hunk 3]
```diff
@@ -71,11 +71,21 @@ In order to develop applications, particularly with the Swift Package Manager, y
...
 The [Windows Package Manager](https://docs.microsoft.com/windows/package-manager/) can be found in the [App Store](https://www.microsoft.com/en-us/p/app-installer/9nblggh4nns1) or be [installed directly](ms-appinstaller:?source=https://aka.ms/getwinget).
 
+#### Install using Scoop
+
+~~~ pwsh
+# Optional: Needed to run a remote script the first time
+Set-ExecutionPolicy RemoteSigned -Scope CurrentUser
+# Command for installing scoop
+Invoke-RestMethod -Url 'get.scoop.sh' | Invoke-Expression
+~~~
```
Comment (compnerd) [inline getting-started/_installing.md:81]: This is confusing here.  These are the instructions for installing using the Windows Package Manager.  Perhaps we should clarify that you can install these tools using documentation pointers?

**Review by compnerd — CHANGES_REQUESTED** [review]

File: getting-started/_installing.md (hunk 1 above)
Comment (Zliced13) [reply getting-started/_installing.md:132]: Yes, my bad.

File: getting-started/_installing.md (hunk 3 above)
Comment (Zliced13) [reply getting-started/_installing.md:81]: Yes, I should have clarified that in the document. I'll fix it as soon as I can.

Comment (Zliced13) [reply getting-started/_installing.md:81]: It looks fixed already, looks good here.

**Comment by MaxDesiatov** [comment]
@swift-ci test

**Comment by kaishin** [comment]
@swift-ci test

**Pull request merged**